docs/api/incoming-message.md

```diff
@@ -80,3 +80,11 @@ An `Integer` indicating the HTTP protocol major version number.
 An `Integer` indicating the HTTP protocol minor version number.
 
 [event-emitter]: https://nodejs.org/api/events.html#events_class_eventemitter
+
+### `response.rawHeaders`
```

Comment:
It looks like this maps directly to what's in the [Node.js](https://nodejs.org/api/http.html#messagerawheaders) docs - I'd like to see an example here but generally speaking given that this improves alignment between Node.js and Electron i think it'd be fine to proceed with this as-is!

Reply:
I've added the same example as found in the Node.js docs.